controllers/articles.controllers.js

```diff
@@ -0,0 +1,14 @@
+const { fetchArticleById } = require("../models/articles.models")
+
+function getArticleById(request, response, next) {
+    const { article_id } = request.params
+    fetchArticleById(article_id)
+        .then((article) => {
+            response.status(200)
+            .send({ article: article })
```

Comment:
If you want to send the data back on the same key, you can just do  ` .send({ article })` and it will auto add the article key 🔑